src/main/java/server/FileResourceHandler.java

```diff
@@ -33,7 +39,8 @@ public boolean delete(String resourceIdentifier) {
     }
 
     public String[] directoryContent() {
-        return new String[0];
+        File rootDirectory = new File("src/main/resources/");
```

Comment:
Is this always going to be the same path? Would you ever want a user to be able to configure this? If nothing else it should probable be a const.

Reply:
In retrospect, we shouldn't be able to have a FileResourceHandler without a rootResourcePath. Made it a constant that lives in Config, and injected it!